src/Preflight/Preflight.php

```diff
@@ -206,6 +206,8 @@ public function loadSymfonyCompatabilityAutoloader(): ClassLoader
             4 => 'v4',  // Drupal 9
             5 => 'v4',  // Early Drupal 10 (Symfony 5 works with Symfony 4 classes, so we don't keep an extra copy)
             6 => 'v6',  // Drupal 10
+            // @TODO Revisit if needed.
+            7 => 'v6',  // Drupal 11
```

Comment:
Should point to v7, interface is different for v6 vs v7. Could probably remove this mechanism if Drush only supported one major version of Drupal at a time, but it seems useful to support multiple versions.

Reply:
I can do this as follow-up work if you want to merge like this to get Symfony 7 into 13.x-dev